fastn-js/js/dom.js

```diff
@@ -960,6 +962,11 @@ class Node2 {
             this.attachAttribute("loading", staticValue);
         } else if (kind === fastn_dom.PropertyKind.Src) {
             this.attachAttribute("src", staticValue);
+        } else if (kind === fastn_dom.PropertyKind.ImageSrc) {
+            const lightSrc = staticValue.get('light');
+            this.attachAttribute("src", lightSrc.get());
```

Comment:
```suggestion
            this.attachAttribute("src", fastn_utils.getStaticValue(lightSrc));
```

Reply:
You have to also add it in closure of `ftd.dark_mode` so when the user switch among light/dark/system mode, it'll render the corresponding image.